pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -36,10 +37,18 @@ type EventListenerSpec struct {
 //
 // +k8s:deepcopy-gen=true
 type Trigger struct {
```

Comment:
Since the `Trigger` type is changing, could you also update the `/test/builder` package?

Reply:
Sure.

Reply:
I would create a separate pr may be for this. It has become too big.